src/NosCore.Handler/DefaultPacketHandler.cs

```diff
@@ -232,6 +236,51 @@ public void Guri(GuriPacket guriPacket)
                 }
             }
         }
+
+        /// <summary>
+        /// say packet
+        /// </summary>
+        /// <param name="sayPacket"></param>
+        public void Say(SayPacket sayPacket)
+        {
+            string message = sayPacket.Message;
+            byte type = sayPacket.Type;
+
+            if (Session.Account.Authority == AuthorityType.Moderator)
+            {
+                type = 12;
```

Comment:
Why not using an enum ? This might not seem useful for now, but it will make the code much clearer later. In OpenNos, there are so many random values where you have to guess what it is